classifier.py

```diff
@@ -64,6 +64,7 @@ class Classifier(object):
 
     GUTENBERG_BOOKSHELF = "gutenberg:bookshelf"
     TOPIC = "schema:Topic"
+    TOPIC_TERM = "Topic term"
```

Comment:
If you classify all your books under `Subject`s of type TOPIC_TERM, but don't provide a way to map TOPIC_TERM classifications to `Genre`s, then your books won't be classified, even in terms of fiction/nonfiction, which means they probably won't show up in the lanes. In most cases I would recommend 

1. File your subjects as TAG instead of making up a new classification scheme. TAG is for free-text classifications and it should also work on English-language topic names.
2. Define an ODILO_TOPIC constant, and define a subclass of TagClassifier that handles it. If TagClassifier is good enough for your purposes, you don't need any code in the subclass, but this will keep your controlled vocabulary separate from other peoples'. (Assuming this is in fact a controlled vocabulary.)
3. If your topic terms are in Spanish, then you'll need to write the classification code yourself. It might be smart to refactor `TagClassifier` and do a Spanish version of the same code.

If you can show me a list of all your topic terms I can give more specific advice. You don't need to have the classifier in this branch but I want the `Subject`s you create to have the right `type` from the start.

Reply:
We considered that and by the moment decided to use the first solution you suggest, just file our subjects as TAG.
Maybe in the future we could offer the BISAC subject code through Connector API withint the record metadata.